solidity/test/relay/LightRelayMaintainerProxy.test.ts

```diff
@@ -0,0 +1,101 @@
+/* eslint-disable no-underscore-dangle */
+/* eslint-disable @typescript-eslint/no-unused-expressions */
+
+import { ethers, helpers, waffle } from "hardhat"
+import { SignerWithAddress } from "@nomiclabs/hardhat-ethers/signers"
+import { expect } from "chai"
+import { ContractTransaction } from "ethers"
+import bridgeFixture from "../fixtures/bridge"
+import type {
+  LightRelayStub,
+  LightRelayMaintainerProxy,
+  ReimbursementPool,
+} from "../../typechain"
+
+import { concatenateHexStrings } from "../helpers/contract-test-helpers"
+
+import headers from "./headersWithRetarget.json"
+import reorgHeaders from "./headersReorgAndRetarget.json"
+import longHeaders from "./longHeaders.json"
+
+const { createSnapshot, restoreSnapshot } = helpers.snapshot
+
+const genesisBlock = headers.oldPeriodStart
+const genesisHeader = genesisBlock.hex
+const genesisHeight = genesisBlock.height // 552384
+const genesisEpoch = genesisHeight / 2016 // 274
+
+const nextEpochStart = headers.chain[9]
+const nextStartHeader = nextEpochStart.hex
+const nextEpochHeight = nextEpochStart.height // 554400
+
+const genesisDifficulty = 5646403851534
+const nextDifficulty = 5106422924659
+
+const proofLength = 4
+
+const fixture = async () => {
+  let deployer: SignerWithAddress
+  let governance: SignerWithAddress
+  let thirdParty: SignerWithAddress
+  let reimbursementPool: ReimbursementPool
+
+  // eslint-disable-next-line @typescript-eslint/no-extra-semi
+  ;({ deployer, governance, thirdParty, reimbursementPool } =
+    await waffle.loadFixture(bridgeFixture))
+
+  console.log("deployer", deployer.address)
+  console.log("governance", governance.address)
+
+  const Relay = await ethers.getContractFactory("LightRelayStub")
+  const relay = await Relay.connect(deployer).deploy()
+  await relay.deployed()
+
+  const MaintainerProxy = await ethers.getContractFactory(
+    "LightRelayMaintainerProxy"
+  )
+  const relayMaintainerProxy = await MaintainerProxy.connect(deployer).deploy(
+    relay.address,
+    reimbursementPool.address
+  )
+  await relayMaintainerProxy.deployed()
+
+  await relay.connect(deployer).transferOwnership(governance.address)
+  await relayMaintainerProxy
+    .connect(deployer)
+    .transferOwnership(governance.address)
```

Comment:
Let's add deployment and setup of this contract to deployment scripts and then eliminate it from here.

Reply:
Added deployment scripts in 9837cd52071e7abca123abdaac653a6c8ac5cdee.
Now the contracts in this tests are deployed by those scripts.